server/configuration.go

```diff
@@ -247,12 +247,18 @@ func (ep *EntryPoints) Set(value string) error {
 		whiteListSourceRange = strings.Split(result["WhiteListSourceRange"], ",")
 	}
 
+	proxyprotocol := false
+	if len(result["ProxyProtocol"]) > 0 {
+		proxyprotocol = strings.EqualFold(result["ProxyProtocol"], "enable") || strings.EqualFold(result["ProxyProtocol"], "on")
```

Comment:
You check to see if `proxyprotocol` is `enable` or `on`, but not `true` based on your example in the toml?

Reply:
Yeah, I just copied what was already done for `Compress` flag. Added `true for both :)